src/app.ts

```diff
@@ -91,4 +91,35 @@ app.get('/flights', (req, res) => {
   res.json(flights);
 });
 
+// /airport
+// Retrieve a list of details for a given airport
+app.get('/airport', (req, res) => {
+  const codeFormatText = 'AAA';
```

Comment:
Nice job reusing this, but we should probably just make the error string once and reuse _that_ instead
```suggestion
  const queryFormattingHelp = 'Airport 'code' is a required parameter and must use a 3 character code such as DFW';
```